src/request.ts

```diff
@@ -590,8 +599,8 @@ class DatastoreRequest {
       void;
   runQuery(query: Query, callback: RunQueryCallback): void;
   runQuery(
-      query: Query, optionsOrCallback?: RunQueryOptions|RunQueryCallback,
-      cb?: RunQueryCallback): void|Promise<RunQueryResponse> {
+      query: Query, optionsOrCallback?: (RunQueryOptions|RunQueryCallback),
```

Comment:
same feedback here and below, you don't need the wrappin parens

Reply:
Sounds good!